static/scss/components/whatsnew.scss

```diff
@@ -48,12 +48,13 @@ header .user-area:not(.arbitrary-class) .c-whatsnew-trigger {
     $arrow-height: 12px;
     display: none;
     position: absolute;
-    transform: translateX(-50%) translateY(calc(50% + #{$spacing-md}));
+    transform: translateX(-50%) translateY(calc(50% + #{$spacing-lg}));
     color: $color-black;
     background-color: $color-white;
     box-shadow: $box-shadow-sm;
     border-radius: $border-radius-sm;
     width: $content-sm;
+    padding-top: $spacing-md;
```

Comment:
This seems to make it seem a bit unbalanced, to me? No biggie, just wondering if it was added intentionally.

This is what it looks like for me now:

![image](https://user-images.githubusercontent.com/4251/156753474-6557eda1-c8a2-47b3-9c22-95a0a7200653.png)